containers/dotnet-postgres/.devcontainer/Dockerfile

```
@@ -0,0 +1,14 @@
+# [Choice] .NET version: 6.0, 5.0, 3.1
+ARG VARIANT="6.0"
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}-focal
```

Similarly, you can update this as follows if bullseye works:
```suggestion
FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
```